creme/stream.py

```diff
@@ -112,6 +116,42 @@ def iter_pandas(X, y=None, **kwargs):
         yield x, yi
 
 
+def iter_vaex(X, y=None, features=None, **kwargs):
+    """Yields rows from a ``vaex.DataFrame``.
+
+    Parameters:
+        X (vaex.DataFrame): A vaex DataFrame housing the training featuers.
+        y (string or vaex.Expression): The column or expression containing the target variable.
+        features (list of strings or vaex.Expressions): A list of features used for training.
+        If None, all columns in ``X`` will be used. Features specifying in ``y`` are ignored.
+
+    Yields:
+        tuple: A pair (``x``, ``y``) where ``x`` is a dict of features and ``y`` is the target.
+    """
+
+    if VAEX_INSTALLED is not True:
```

Comment:
There's many ways to do this, and indeed we used this same kind of check for `pandas`. What I would prefer here is to place the `from vaex.utils import _ensure_strings_from_expressions, _ensure_list` import inside the `iter_vaex` function. This way an `ImportError` will be raised if `vaex` isn't installed.

Reply:
Hey! 
Sure, what you say makes sense, in fact that was my 1st version.. but than my local linter started to complain..  :) I will make adjustments per your suggestion.